osbuild/util/parsing.py

```diff
@@ -72,9 +72,9 @@ def parse_input(url: ParseResult, args: Dict) -> os.PathLike:
     return root
 
 
-def parse_location(location: str, args: Dict) -> str:
+def parse_location_ext(location: str, args: Dict) -> Tuple[str, str]:
```

Comment:
what does `parse_location_ext` mean exactly? Maybe something more self-explanatory like `parse_location_into_parts()` would be better?

Reply:
`_ext` stands for extended, wasn't able to find better name. Thx for the advise!